gazebo_plugins/src/gazebo_ros_wheel_slip.cpp

```diff
@@ -75,25 +89,63 @@ void GazeboRosWheelSlip::Load(gazebo::physics::ModelPtr _model, sdf::ElementPtr
     [this](std::vector<rclcpp::Parameter> parameters) {
       auto result = rcl_interfaces::msg::SetParametersResult();
       result.successful = true;
+
       for (const auto & parameter : parameters) {
         std::string param_name = parameter.get_name();
+
+        if (param_name.find("slip_compliance_unitless_lateral/") != std::string::npos) {
+          auto wheel_name = param_name.substr(param_name.find("/") + 1);
+          if (impl_->mapSlipLateralDefault.count(wheel_name) == 1) {
+            // The parameter has a valid name
+            double slip = parameter.as_double();
+            if (slip >= 0.) {
+              RCLCPP_INFO(
+                impl_->ros_node_->get_logger(),
+                "New lateral slip compliance for %s: %.3e", wheel_name.c_str(), slip);
+              this->SetSlipComplianceLateral(wheel_name, slip);
+            } else {
+              result.successful = false;
+              result.reason = "Slip compliance values cannot be negative";
+            }
+          }
+        }
+
+        if (param_name.find("slip_compliance_unitless_longitudinal/") != std::string::npos) {
+          auto wheel_name = param_name.substr(param_name.find("/") + 1);
+          if (impl_->mapSlipLongitudinalDefault.count(wheel_name) == 1) {
+            // The parameter has a valid name
+            double slip = parameter.as_double();
+            if (slip >= 0.) {
+              RCLCPP_INFO(
+                impl_->ros_node_->get_logger(),
+                "New longitudinal slip compliance for %s: %.3e", wheel_name.c_str(), slip);
+              this->SetSlipComplianceLongitudinal(wheel_name, slip);
+            } else {
+              result.successful = false;
+              result.reason = "Slip compliance values cannot be negative";
+            }
+          }
+        }
+
         if (param_name == "slip_compliance_unitless_lateral") {
           double slip = parameter.as_double();
           if (slip >= 0.) {
             RCLCPP_INFO(
               impl_->ros_node_->get_logger(),
-              "New lateral slip compliance: %.3e", slip);
+              "New lateral slip compliance for all wheels: %.3e", slip);
```

Comment:
I guess this log statement is redundant with the RCLCPP_INFO in the ParameterEventHandler callback. Maybe we can remove one of them.